app/pages/project/instances/instance/tabs/NetworkingTab.tsx

```diff
@@ -77,14 +85,43 @@ const SubnetNameFromId = ({ value }: { value: string }) => {
   return <span className="text-secondary">{subnet.name}</span>
 }
 
-function ExternalIpsFromInstanceName({ value: primary }: { value: boolean }) {
+function IpLink({ ip }: { ip: string }) {
+  return (
+    <a
+      className="underline text-sans-semi-md text-secondary hover:text-default"
+      href={ip}
```

Comment:
This needs a protocol – we can assume this is `http` right?

Reply:
Not sure. Might be better to do https and let them fix it after nav if it’s wrong.